OpenLIFUPrePlanning/OpenLIFUPrePlanning.py

```diff
@@ -558,27 +558,18 @@ def virtual_fit(
 
         add_slicer_log_handler("VirtualFit", "Virtual fitting")
 
-        # TODO: Many quantities are hard-coded here will not have to be when these two issues are done:
-        # https://github.com/OpenwaterHealth/OpenLIFU-python/issues/166
-        # https://github.com/OpenwaterHealth/OpenLIFU-python/issues/165
+        transducer_openlifu : "openlifu.Transducer" = transducer.transducer.transducer
+        protocol_openlifu : "openlifu.Protocol" = protocol.protocol
+
+        units = "mm" # These are the units of the output space of the transform returned by get_IJK2RAS
+
         vf_transforms = openlifu_lz().virtual_fit(
```

Comment:
Here you can see that the virtual_fit now consumes options from the protocol, and consumes standoff_transform from the transducer. 

@sadhana-r For this review, it is enough to just check that when you install the correct version of openlifu python and pull the dvc data that comes with that version, that the virtual fitting runs smoothly. Thanks!